SmartDeviceLink/SDLStreamingMediaManager.m

```diff
@@ -416,7 +416,7 @@ - (BOOL)sdl_configureVideoEncoderWithError:(NSError *__autoreleasing *)error {
 
     if (status != noErr) {
         // TODO: Log the error
-        if (*error != nil) {
+        if (error != nil) {
```

Comment:
These should be `if (error != NULL)` because we are comparing a reference pointer, not an Objective-C object.